tools/Makefile

```diff
@@ -57,7 +56,6 @@ $(curdir)/gmp/compile := $(curdir)/libtool/compile
 $(curdir)/isl/compile := $(curdir)/gmp/compile
 $(curdir)/libressl/compile := $(curdir)/pkgconf/compile
 $(curdir)/libtool/compile := $(curdir)/m4/compile $(curdir)/autoconf/compile $(curdir)/automake/compile $(curdir)/missing-macros/compile
-$(curdir)/lzma-old/compile := $(curdir)/zlib/compile
```

Comment:
Is `lzma-old` still used by anything after these changes?

Reply:
think it's used for the squashfs3 that from what i can see needs to be kept while i find an user that can test my hack idea for the fake squashfs3

Reply:
It looks like `lzma-old` is only used by the `squashfs` application. The new one probably uses the version of the lzma library which is still developed.